src/main/java/by/itech/lab/supplier/dto/UserDto.java

```diff
@@ -0,0 +1,78 @@
+package by.itech.lab.supplier.dto;
+
+import by.itech.lab.supplier.domain.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.apache.catalina.authenticator.Constants;
+
+import javax.persistence.*;
+import javax.validation.constraints.*;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+import java.time.Instant;
+import java.time.LocalDate;
+import java.util.Date;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+public class UserDto implements BaseDto {
+
+
+    public UserDto(User user) {
```

Comment:
![image](https://user-images.githubusercontent.com/44436165/96954449-187b9b00-14fc-11eb-8bb3-e8e7b6af3c30.png)
Why just not use mapper? Why do you bound your repo and UI levels?